src/resources/styles.ts

```diff
@@ -3,7 +3,7 @@ import { css } from "lit";
 export const darkStyles = {
   "primary-background-color": "#111111",
   "card-background-color": "#1c1c1c",
-  "secondary-background-color": "#202020",
+  "secondary-background-color": "#282828",
```

Comment:
To match the "color delta" from the light theme and have a visual separation (between `primary-background-color` and `secondary-background-color`).

Light theme uses `#ff` and `#e5` which means a delta of 11.

With `#28` we now have the same distance to `#11`.